common/sketch/src/main/java/org/apache/spark/util/sketch/CountMinSketch.java

```diff
@@ -55,6 +57,25 @@
  */
 abstract public class CountMinSketch {
   /**
+   * Version number of the serialized binary format.
+   */
+  public enum Version {
```

Comment:
This should be count min sketch specific, because the two binary protocols can and should evolve separately.